components/developer/golang-120/golang-120.p5m

```diff
@@ -0,0 +1,11495 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2023 Till Wegmueller
+#
+
+set name=pkg.fmri \
+    value=pkg:/$(COMPONENT_FMRI)-119@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
```

Comment:
You'll need to fix the fmri here. Probably better move the complete definition to the Makefile.

Reply:
Fixed